src/puppetlabs/puppetdb/utils.clj

```diff
@@ -257,3 +257,10 @@
       {:href (to-href data)}
       (-> (sutils/parse-db-json data)
           (update :href to-href)))))
+
+(defn hsql?
+  "given the globals configuration bundle, determine if hsqldb is being used."
+  [globals]
```

Comment:
db-spec means something specific to me in the context of jdbc. It's the classname, subprotocol, subname map formed by the [database] section of our config. globals is our name for a different map that we use in a lot of places, and I think it'd be confusing to call it db-spec here.